### DISCUSSION TITLE
Proxmox Backup Server LXC mount NFS

### DISCUSSION THREAD

**Discussion by jlai79** [tteck/Proxmox#3852] [Help Needed]
I just installed PBS and am now trying to mount an NFS share on my Synology NAS. I'm still new to Proxmox, but I found out that restricted LXC containers cannot mount NFS unless explicitly enabled.

So I mounted my /volume1/backup to /mnt/nasbackup on the Proxmox host and added this to my lxc config:
lxc.mount.entry = /mnt/nasbackup/ nfs mnt/nasbackup/ none bind 0 0

/mnt/nasbackup is chown backup:backup and chmod 775 

On start I get this error:
mount_entry: 2213 No such file or directory - Failed to mount "/mnt/backup/" on "/usr/lib/x86_64-linux-gnu/lxc/rootfs/nfs"
lxc_setup: 3915 Failed to setup mount entries
do_start: 1273 Failed to setup container "107"
sync_wait: 34 An error occurred in another process (expected sequence number 3)
__lxc_start: 2114 Failed to spawn container "107"
startup for container '107' failed

What's wrong here? Am I missing something?

**Comment by MorphyDK** [comment]
You need to add the mount in the PBS shell - not the host.
Just setup my own setup - works just fine :)

**Reply by jlai79** [reply]
Mounting in PBS was the first thing I tried:

mount -t nfs -o soft 192.168.17.5:/volume1/backup /mnt/backup
mount.nfs: Operation not permitted

That's why I searched for the reason and found this:
[Mount NFS – Operation Not Permitted In Proxmox container | Resolved](https://bobcares.com/blog/mount-nfs-operation-not-permitted-in-proxmox-container/)

**Reply by MorphyDK** [reply]
You need to install nfs-common first

**Reply by jlai79** [reply]
nfs-common is already installed

**Reply by MorphyDK** [reply]
Have you ticked nfs in options of the container? Im running priviledged container ( as I recall )

**Reply by jlai79** [reply]
No. The script by @tteck creates an unprivileged container. In privileged containers there's no problem with nfs mounts.

**Reply by MorphyDK** [reply]
Well then im lost 😊

**Comment by havardthom** [comment]
Goto tutorial for mounting shares in unprivileged LXCs: https://forum.proxmox.com/threads/tutorial-unprivileged-lxcs-mount-cifs-shares.101795/

**Reply by jlai79** [reply]
Does this tutorial apply to NFS as well, or only to CIFS?
